com.unity.netcode.gameobjects/Runtime/NetworkVariable/Collections/NetworkDictionary.cs

```diff
@@ -308,17 +251,25 @@ public override bool IsDirty()
         }
 
         /// <inheritdoc />
-        public ICollection<TKey> Keys => m_Dictionary.Keys;
+        public bool TryGetValue(TKey key, out TValue value)
+        {
+            return m_Dictionary.TryGetValue(key, out value);
+        }
 
         /// <inheritdoc />
-        public ICollection<TValue> Values => m_Dictionary.Values;
+        public NativeArray<TKey> Keys(Allocator alloc)
+        {
+            return m_Dictionary.GetKeyArray(alloc);
+        }
 
         /// <inheritdoc />
-        public int Count => m_Dictionary.Count;
+        public NativeArray<TValue> Values(Allocator alloc)
+        {
+            return m_Dictionary.GetValueArray(alloc);
+        }
 
         /// <inheritdoc />
-        // TODO: remove w/ native containers
-        public bool IsReadOnly => m_Dictionary.IsReadOnly;
```

Comment:
This was an IDictionary hold over, we didn't do anything with it really before.  We could I guess add a read-only mode flag later and then block writes